src/Database/Driver/Sqlite.php

```diff
@@ -32,6 +32,11 @@ class Sqlite extends Driver
 {
     use SqliteDialectTrait;
 
+    /**
+     * @var int|null Maximum alias length or null if no limit
+     */
+    protected const MAX_ALIAS_LENGTH = null;
```

Comment:
Instead of this we can just add a default `getMaxAliasLength()` implementation in `Driver` class which returns `null`.